cpp/valgrind.supp

```diff
@@ -82,3 +82,15 @@
     fun:curl_share_init
     fun:*Azure*CurlConnection*
 }
+{
+    <protobuf>:leak on library initialization
+    Memcheck:Leak
+    ...
+    fun:*google*protobuf*AddDescriptors*
```

Comment:
It's probably not a big deal, but why don't we use a little more canonical symbol names like `fun:*google*protobuf*AddDescriptorsRunner*` and `fun:*google*protobuf*InternalRegisterGeneratedFile*`?

Reply:
I typed those by hand and was a bit silly. Now fixed.